Dependencies/xr/CMakeLists.txt

```diff
@@ -12,6 +12,8 @@ elseif (IOS)
     set(SOURCES ${SOURCES}
         "Source/ARKit/XR.mm")
 else()
+    # Avoid picking up system installed jsoncpp in favor of source distributed with openxr_loader project
+    set(BUILD_WITH_SYSTEM_JSONCPP OFF CACHE BOOL "disable using system installed jsoncpp" FORCE)
```

Comment:
You probably don't want to use FORCE here since that will prevent users from using a custom one if they really wanted to.

Reply:
We can try not using force, my concern is that removing force would cause this value to get overwritten in the jsoncpp setup

Reply:
please avoid checking this in, ill push a commit to test whether force is needed

Reply:
looks like ci succeeded without the force flag, ill go ahead and update this once the build completes